ui/StatusQ/src/StatusQ/Components/StatusDateGroupLabel.qml

```diff
@@ -15,16 +15,15 @@ StatusBaseText {
     horizontalAlignment: Text.AlignHCenter
 
     text: {
-        const currentMsgDate = new Date(messageTimestamp)
-        const prevMsgDate = new Date(previousMessageTimestamp)
+        const lastMessageInDays = Math.floor(previousMessageTimestamp / 86400000)
```

Comment:
Would be nice to name this magic number somehow `const xyz = 86400000`

Reply:
Done